doc/source/performance/procfs-plugin.rst

```diff
@@ -0,0 +1,83 @@
+.. procfs-plugin:
+
+======================================================================
+Using the ProcFS plugin
+======================================================================
+
+
+Implemented in |Percona Server| 8.0.25-15, the ProcFS plugin provides access to the Linux performance counters by running SQL queries against a Percona Server for MySQL 8.0 (or other MySQL 8.0-compatible variant). This plugin is useful for Cloud and MySQL-as-a-service installations.
+
+The plugin reads files from /proc and /sys file systems and returns the file name and any content in rows in the `INFORMATION_SCHEMA.PROCFS` view. The schema definition of this view is:
+
```

Comment:
It would be more consistent if we used same template that is used for other I_S tables, like: 
 diagnostics/misc_info_schema_tables.rst

Reply:
Same problem as :variable: - these tables will be reformatted so that we can remove psdom.

Reply:
Added link to I_S tables doc.

Reply:
"in some environments" repeated twice in same sentence.

Reply:
Fixed.